model/src/main/java/com/plhal/ares/dbLayer/DataRepositoryImpl.java

```diff
@@ -111,12 +118,14 @@ public Firma najdiFirmu(String ico) {
 
                 tempSO = StatutarniOrgan.builder()
                         .prijmeni(prijmeni)
+                        .statutarniOrganId(ico + i)
```

Comment:
Tomuto uplne nerozumim. Co ma ICO spolecneho s cleny statutaniho organu. Presneji proc je jejich ico o 1 vyssi?

Reply:
Tohle jsem tam udelal z toho duvodu, ze kdyz jsem zavolal metodu firmaRepository.save(firma) nad firmou, ktera uz v db je (tedy se provedl update), tak se mi statutarni organ vlozil do tabulky duplicitne (primary key u statutarniho organu byl auto increment). Tak jsem zrusil auto increment a id generoval timto zpusobem. Duplicity uz jsem tam pri volani metody firmaRepository.save(firma) nemel. Rozumim, ze to nebude dobre reseni..